values/velero/velero.gotmpl

```diff
@@ -35,68 +42,93 @@ initContainers:
     volumeMounts:
       - mountPath: /target
         name: plugins
-{{- end }}
 
 podSecurityContext:
   runAsUser: 1000
 
-serviceMonitor:
-    enabled: true
-    namespace: velero
-
 logLevel: {{ $vl.logLevel }}
 
 configuration:
-  {{- with $provider | get "azure" nil }}
+  {{- if eq $sp "azureBlob" }}
   provider: azure
+  backupStorageLocation:
+    name: otomi
+    default: true
+    bucket: {{ $storage.provider.azureBlob.bucket }}
+    config:
+      storageAccount: {{ $storage.provider.azureBlob.storageAccount }}
+      resourceGroup: {{ $storage.provider.azureBlob.resourceGroup }}
   {{- end }}
-  {{- with $provider | get "aws" nil }}
+  {{- if eq $sp "s3" }}
```

Comment:
$sp.type ?

Reply:
More or less the same. $sp is type here. Changed in favor of fewer characters used ;)